templates/workspaces/base/porter.yaml

```diff
@@ -96,7 +96,7 @@ parameters:
   - name: app_service_plan_sku
     type: string
     description: "The SKU used when deploying an Azure App Service Plan"
-    default: "P1v3"
+    default: "P1v2"
```

Comment:
I thought that this would need to be a default of P1v3 - as that is what we want customers to use

Reply:
Uh oh, this slipped in from my testing! Thanks for the catch